application/gqlgen/gqlgen.yml

```diff
@@ -47,6 +47,8 @@ models:
     fields:
       id:
         resolver: true
+      neededBefore:
+        resolver: true
```

Comment:
I'm just curious. Did you try it without a custom resolver?

Reply:
If I don't do that, I get this ...
```
gqlgen/post.go:17:23: cannot use &postResolver literal (type *postResolver) as type PostResolver in return argument:
	*postResolver does not implement PostResolver (missing NeededBefore method)
```
Is there something else I need to add/remove, do you think?

Reply:
Looks like its logic automatically indicates that a resolver is needed. To get that error, did you also remove your `NeededBefore` resolver?